tests/unit/test_attribute_mapping.py

```diff
@@ -37,6 +45,17 @@ def test_resource(self):
         expected = self.mappings_json["unifiedAttributeName"]
         self.assertEqual(expected, test.unified_attribute_name)
 
+    @responses.activate
+    def test_delete(self):
+        general_url = (
+            "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings"
+        )
+        specific_url = "http://localhost:9100/api/versioned/v1/projects/4/attributeMappings/19629-12"
```

Comment:
You could base this off of `general_url` (`specific_url = general_url + "/19629-12"`)